Check broken links (#4)
christinerose/ocaml.org: data/planet/ocamlorg/turn-your-editor-into-an-ocaml-ide.md

handles the task of indenting your OCaml files.  It is an OCaml executable that
can be used from the command line or directly from your editor.

> The second tool, [merlin](http://the-lambda-church.github.io/merlin/ - [404 Not Found]) performs

Reviewer: ```suggestion
The second tool, [Merlin](https://github.com/ocaml/merlin) performs
```

> "static analysis" of your source files.  The analysis is then used to provide error reporting, source

Reviewer: ```suggestion
"static analysis" of your source files. The analysis is then used to provide error reporting, source
```

> To maintain synchronization with the compiler, merlin needs some information

Reviewer: ```suggestion
To maintain synchronisation with the compiler, Merlin needs some information
```

about the structure of your project: build and source directories, package
dependencies, syntax extensions.  This structure can be described in a `.merlin` file in the root directory of your project.

> The [`.merlin` file for the merlin project](https://github.com/the-lambda-church/merlin/blob/master/.merlin - [404 Not Found]) illustrates the syntax.

Reviewer: ```suggestion
The [`.merlin` file for the Merlin project](https://github.com/ocaml/merlin) illustrates the syntax.
```
